demos/sdk_output_skeleton_1_ghe/run/baseline_scenario/ghe_dir/sys_params_detailed_geo.json

```diff
@@ -0,0 +1,149 @@
+        "fan_design_head": 150
+      },
+      "photovoltaic_panels": [],
+      "diesel_generators": [],
+      "battery_banks": []
+    },
+    {
+      "geojson_id": "9",
+      "load_model": "time_series",
+      "load_model_parameters": {
+        "time_series": {
+          "filepath": "C:\\Users\\tcharan\\Desktop\\NRELWork\\Gitrepos\\geojson-modelica-translator\\tests\\management\\data\\sdk_project_scraps\\run\\baseline_scenario\\5\\016_export_modelica_loads\\modelica.mos",
+          "delta_temp_air_cooling": 10,
+          "delta_temp_air_heating": 18,
+          "has_liquid_cooling": true,
+          "has_liquid_heating": true,
+          "has_electric_cooling": false,
+          "has_electric_heating": false,
+          "max_electrical_load": 45830800,
+          "temp_chw_return": 12,
+          "temp_chw_supply": 7,
+          "temp_hw_return": 35,
+          "temp_hw_supply": 40,
+          "temp_setpoint_cooling": 24,
+          "temp_setpoint_heating": 20
+        }
+      },
+      "ets_model": "Fifth Gen Heat Pump",
+      "fifth_gen_ets_parameters": {
+        "supply_water_temperature_building": 15,
+        "chilled_water_supply_temp": 5,
+        "hot_water_supply_temp": 50,
+        "cop_heat_pump_heating": 2.5,
+        "cop_heat_pump_cooling": 3.5,
+        "ets_pump_flow_rate": 0.0005,
+        "ets_pump_head": 10000,
+        "fan_design_flow_rate": 0.25,
+        "fan_design_head": 150
+      }
+    }
+  ],
+  "district_system": {
+    "fifth_generation": {
+      "soil": {
+        "conductivity": 2.0,
+        "rho_cp": 2343493,
+        "undisturbed_temp": 18.3
+      },
+      "horizontal_piping_parameters": {
+        "hydraulic_diameter": 0.089,
+        "insulation_thickness": 0.2,
+        "insulation_conductivity": 2.3,
+        "diameter_ratio": 11,
+        "roughness": 1e-06,
+        "rho_cp": 2139000,
+        "number_of_segments": 1,
+        "buried_depth": 1.5
+      },
+      "central_pump_parameters": {
+        "pump_design_head": 60000,
+        "pump_flow_rate": 0.01
+      },
+      "ghe_parameters": {
+        "version": "0.2.5",
+        "ghe_dir": "tests\\management\\data\\sdk_project_scraps\\run\\baseline_scenario\\ghe_dir",
+        "fluid": {
+          "fluid_name": "Water",
+          "concentration_percent": 0.0,
+          "temperature": 20
+        },
+        "grout": {
+          "conductivity": 1.0,
+          "rho_cp": 3901000
+        },
+        "pipe": {
+          "inner_diameter": 0.0216,
+          "outer_diameter": 0.0266,
+          "shank_spacing": 0.0323,
+          "roughness": 1e-06,
+          "conductivity": 0.4,
+          "rho_cp": 1542000,
+          "arrangement": "singleutube"
+        },
+        "simulation": {
+          "num_months": 240
+        },
+        "geometric_constraints": {
+          "b_min": 3.0,
+          "b_max": 10.0,
+          "max_height": 135.0,
+          "min_height": 60.0,
+          "method": "rectangle"
```

Comment:
Thanks, @chriswmackey. Shouldn't this be `polygon` then, in this case?

Reply:
I made the change in the test file, @mitchute . But you can see that this property currently has no meaning in the ThernmalNetwork package and I have preserved that lack of meaning in my PR. The thing that triggers the use of the GeoJSON geometry of the GHE is the lack of the `"ghe_geometric_params"` properties. Not the currently unused `method` property.

I can change my PR so that this `method` property has meaning. However, `polygon` is not currently an option that is supported in the system parameter schema:

https://docs.urbanopt.net/geojson-modelica-translator/input_file_documentation.html#system-parameters-schema

![image](https://github.com/user-attachments/assets/4e49dd01-e706-41ac-b940-b09ba20d703b)

Only  `rectangle` and `nearsquare` are supported in the GMT. As I said above, neither of them mean anything so this probably needs to be changed. Maybe this `method` shouldn't even go under the `geometric_constraints` anymore and it should instead go under `ghe_specific_params` next to the `ghe_geometric_params` that provide the inputs for the various alternative methods that don't use the GeoJSON geometry (like Rectangle or NearSquare).

Do you want to send a PR to the GMT to change the system parameter schema so that it works the way that you want?

If you can do that and we can get a re-release of the GMT + URBANopt CLI from @vtnate with the new schema change, I can change my PR here to use the new schema.

Or we can just merge this PR for the time being so that I'm unblocked with testing this GeoJSON capability in Rhino/Grasshopper. Then we can address the schema change with a new set of coordinated PRs. What do you think?